### DISCUSSION TITLE
Youtube comment section

### DISCUSSION THREAD

**Discussion by 4g03fhju** [ReVanced/revanced-discussions#150] [Question] [answered]
It would be nice to have the old section of the comments, I was using Vanced v.14.21.54 just for that.
Is it possible to give the possibility to the user to choose if to have the old or new position of the comments?

**Comment by oSumAtrIX** [comment, marked as answer]
YouTube removed it, it might be difficult

**Comment by fym35** [comment]
that was in original vanced aswell
(they added it in Vanced 15)
